test/src/analyzers/lint_analyzer/rules/rules_list/ban_name/ban_name_rule.dart

```diff
@@ -0,0 +1,49 @@
+import 'package:dart_code_metrics/src/analyzers/lint_analyzer/models/severity.dart';
+import 'package:dart_code_metrics/src/analyzers/lint_analyzer/rules/rules_list/ban_name/ban_name_rule.dart';
+import 'package:test/test.dart';
+
+import '../../../../../helpers/rule_test_helper.dart';
+
+const _examplePath = 'ban_name/examples/example.dart';
+
+void main() {
+  group('BanNameRule', () {
+    test('initialization', () async {
+      final unit = await RuleTestHelper.resolveFromFile(_examplePath);
+      final issues = BanNameRule().check(unit);
+
+      RuleTestHelper.verifyInitialization(
+        issues: issues,
+        ruleId: 'prefer-correct-type-name',
```

Comment:
Interesting... and the test is not failing? Could you fix this, I'll take a look why it's not failing a bit later.